app/jmeter/jira.jmx

```diff
@@ -6848,14 +6848,21 @@ vars.put(&quot;extend_action&quot;, extend_action);
               <boolProp name="TransactionController.parent">true</boolProp>
             </TransactionController>
             <hashTree>
-              <RandomVariableConfig guiclass="TestBeanGUI" testclass="RandomVariableConfig" testname="random page number (1 - 100)" enabled="true">
-                <stringProp name="variableName">page_number</stringProp>
-                <stringProp name="outputFormat"></stringProp>
-                <stringProp name="minimumValue">1</stringProp>
-                <stringProp name="maximumValue">100</stringProp>
-                <stringProp name="randomSeed"></stringProp>
-                <boolProp name="perThread">false</boolProp>
-              </RandomVariableConfig>
+              <BeanShellPreProcessor guiclass="TestBeanGUI" testclass="BeanShellPreProcessor" testname="get project pages count" enabled="true">
+                <boolProp name="resetInterpreter">false</boolProp>
+                <stringProp name="parameters"></stringProp>
+                <stringProp name="filename"></stringProp>
+                <stringProp name="script">import org.apache.commons.io.FileUtils;
+
+int projects = FileUtils.readLines(new File(&quot;datasets/jira/project_keys.csv&quot;)).size();
```

Comment:
It feels like a path to file should be moved somewhere to configuration.

Reply:
Yes, it's a good idea, but let's do it later (we should find the best approach to do it)